build/templates/UmbracoProject/.template.config/template.json

```
@@ -67,6 +77,32 @@
             ],
             "replaces": "net5.0",
             "defaultValue": "net5.0"
+        },
+        "SkipRestore": {
+            "type": "parameter",
+            "datatype": "bool",
+            "description": "If specified, skips the automatic restore of the project on create.",
+            "defaultValue": "false"
+        },
+        "HttpPort": {
+            "type": "generated",
+            "generator": "port",
+            "replaces": "HTTP_PORT_FROM_TEMPLATE",
+            "parameters": {
+                "high": 65535,
+                "low": 1024,
+                "fallback": 9000
```

```suggestion
                "fallback": 5000
```

I think fallback should be 5000 as the default value from ASP.NET Core projects.. 9000 looks like our internal ports